packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -1150,7 +1140,7 @@ const handleEncryptionVersionMigration =
 
         // 2. in general, metadata related actions are per device as everything is encrypted by device.metadata[version].key
         let { device } = getState().suite;
-        if (!device?.state || device.metadata.status !== 'enabled') {
+        if (!device?.state) {
             return { success: false, error: 'metadata migration: device unexpected state' };
```

Comment:
lets think about other places that might requrie toast

Reply:
should be handled better now 
1. removed toast from "atomic action" such as "save labels" otherwise actions scheduled in batch would fire dozens of toasts
2. atomic actions return only success:boolea, error?: string, and toast is dispatched by caller if needed

<img width="628" alt="image" src="https://github.com/trezor/trezor-suite/assets/30367552/87d07f45-2552-4481-9a24-7e9c49cdc003">